src/features/game/expansion/components/Water.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+
+import { GRID_WIDTH_PX } from "features/game/lib/constants";
+
+import { Section } from "lib/utils/hooks/useScrollIntoView";
+
+import dragonfly from "assets/decorations/dragonfly.gif";
+import Shark from "./water/Shark";
+
+import goblinSwimming from "assets/npcs/goblin_swimming.gif";
+import goblinSnorkling from "assets/npcs/goblin_snorkling.gif";
+import swimmer from "assets/npcs/swimmer.gif";
+
+export const Water: React.FC = () => {
+  return (
+    // Container
+    <div
+      style={{
+        height: "inherit",
+        width: "calc(100%)",
+        left: "0",
+        top: "0",
+      }}
+      className="absolute"
+    >
+      {/* Above Land */}
+      <Shark side="top" />
+
+      {/* Below Land */}
+      <Shark side="bottom" />
+
+      {/* Navigation Center Point */}
+      <div className="h-full w-full relative">
+        <span
+          id={Section.Water}
+          className="absolute left-1/2 top-1/2 transform -translate-x-1/2 -translate-y-1/2"
+        />
+
+        <img
+          src={dragonfly}
+          className="absolute left-2/3 top-1/2 animate-float"
```

Comment:
<img width="215" alt="Screen Shot 2022-07-08 at 9 02 06 am" src="https://user-images.githubusercontent.com/11745561/177885952-40f0f3ec-f8b1-4988-8284-431f92fbe8d6.png">

Just another small one, but the dragonfly pixels are much larger than the water pixels